app/src/main/res/values/strings.xml

```diff
@@ -383,6 +384,10 @@
     <string name="empty_group_name_error">Please enter a group name</string>
     <string name="group_name_exceeded_limit_error">Group name should not exceed 64 characters
     </string>
+    <string name="group_can_not_be_created_title">Group can’t be created</string>
+    <string name="group_can_not_be_created_federation_conflict_description">People from backends %1$s can’t join the same group conversation.\n\n To create the group, remove affected participants.</string>
```

Comment:
Could you remove the space before 'To create the group...'
It's not needed if you look at the screenshot

Reply:
done